tests/avro/test_cached_client.py

```diff
@@ -140,19 +142,25 @@ def hash_func(self):
 
     def test_cert_no_key(self):
         with self.assertRaises(ValueError):
-            self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                     cert_location='/path/to/cert')
+            self.client = CachedSchemaRegistryClient({
+                'url': 'https://127.0.0.1:65534',
+                'ssl.certificate.location': '/path/to/cert',
+            })
 
     def test_cert_with_key(self):
-        self.client = CachedSchemaRegistryClient(url='https://127.0.0.1:65534',
-                                                 cert_location='/path/to/cert',
-                                                 key_location='/path/to/key')
+        self.client = CachedSchemaRegistryClient({
+            'url': 'https://127.0.0.1:65534',
+            'ssl.certificate.location': '/path/to/cert',
+            'ssl.key.location': '/path/to/key'
+        })
         self.assertTupleEqual(('/path/to/cert', '/path/to/key'), self.client._session.cert)
 
-    def test_cert_path(self):
```

Comment:
Why was this test removed?

Reply:
Because from https://github.com/confluentinc/confluent-kafka-python/blob/master/src/confluent_kafka/avro/cached_schema_registry_client.py#L151, for `ssl.certificate.location` and `ssl.key.location`,  **# Both values can be None or no values can be None**. The previous test `test_cert_path` only set `ssl.certificate.location`, but doesn't raise any exception.

There is a function above called `test_cert_no_key ` already covered the cases only set `ssl.certificate.location` : https://github.com/confluentinc/confluent-kafka-python/blob/master/tests/avro/test_cached_client.py#L141. So the below test I added `test_key_no_cert` covers the cases that if only set `ssl.key.location`, an exception will raise.